.github/workflows/codeql.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: ["main"]
+    branches: ["release"]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: ["main"]
+    branches: ["release"]
   schedule:
     - cron: "0 0 * * 1"
 
```

Comment:
I think we dropped this in other places, if thats right then we should drop it here as well

Reply:
done